### ISSUE TITLE
PoliceGesture Dataset Training with YOLO

### ISSUE THREAD

**Issue by yildizyakup** [zc402/ctpgr-pytorch#12]
Firstly, thank you for your contribution it really helps when developing similar project. 

I want to train a model that can predict your traffic gestures. When training model I want to use your policagesture dataset. I am considering saving the keypoints in a sequence by using YOLO's pose estimation model. I want to ask how could I train LSTM model by using your labels and my keypoints. 

For example I have run my yolo model on your training data 001.mp4 file I got keypoints. How should I match the corresponding labels?

Is there a specific range between frames. How many frames or how much time internal correspond to gestures that are given in  .csv files? 

I would be very grateful if you could give me an answer. Thank you for your concern

**Comment by zc402** [comment]
It is totally fine to use yolo keypoint estimation and LSTM to predict gestures. My keypoint network was a vgg network, and it is outdated, applying a newer method can improve performance. The label was marked for each frame, from first frame to last frame. I remembered that the framerate was 10fps.

**Comment by yildizyakup** [comment]
Thank you so much
